_summaries/zeiler2012adadelta_1.md

```diff
@@ -1,19 +1,48 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{zeiler2012adadelta}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author:  sritank
+score:  10
 ---
 
 TODO: Summarize the paper:
```

Comment:
Small personal preference for formatting, can remove the "TODO summarize the paper" as well as the other bullets for the template "What is the core idea?" etc etc and replace with more insightful titles/headers. (eg. Main Idea,  Implementation, etc etc)

Reply:
Done